cicecore/drivers/direct/hadgem3/CICE_InitMod.F90

```diff
@@ -71,7 +71,7 @@ subroutine cice_init
       use ice_domain, only: init_domain_blocks
       use ice_domain_size, only: ncat, nfsd
       use ice_dyn_eap, only: init_eap, alloc_dyn_eap
-      use ice_dyn_shared, only: kdyn, init_dyn, basalstress, alloc_dyn_shared
+      use ice_dyn_shared, only: kdyn, init_dyn, seabedstress, alloc_dyn_shared
```

Comment:
This variable is not used in that driver, so we might as well remove it from that line:
```suggestion
      use ice_dyn_shared, only: kdyn, init_dyn, alloc_dyn_shared
```